packages/terra-compact-interactive-list/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 1.0.0 - (January 2, 2024)
```

Comment:
Are we ready to do the full release?

Reply:
Yep - we're ready to release compact-interactive-list!

Reply:
I discussed that with @chrismichalewicz and the team before the holidays, the answer was yes.